articles/partner-twilio-php-make-phone-call.md

```diff
@@ -24,76 +24,80 @@ The following example shows you how you can use Twilio to make a call from a PHP
 
 You'll need to do the following to use the code in this topic:
 
-1. Acquire a Twilio account and authentication token. To get started with Twilio, evaluate pricing at [http://www.twilio.com/pricing][twilio_pricing]. You can sign up for a trial account at [https://www.twilio.com/try-twilio][try_twilio]. For information about the API provided by Twilio, see [http://www.twilio.com/api][twilio_api].
+1. Acquire a Twilio account and authentication token from your [Twilio Console][twilio_console]. To get started with Twilio, evaluate pricing at [http://www.twilio.com/pricing][twilio_pricing]. You can sign up for a trial account at [https://www.twilio.com/try-twilio][try_twilio]. For information about the API provided by Twilio, see [http://www.twilio.com/api][twilio_api].
 2. Obtain the [Twilio library for PHP](https://github.com/twilio/twilio-php) or install it as a PEAR package. For more information, see the [readme file](https://github.com/twilio/twilio-php/blob/master/README.md).
-3. Install the Azure SDK for PHP. For an overview of the SDK and instructions on installing it, see [Set up the Azure SDK for PHP][setup_php_sdk].
+3. Install the Azure SDK for PHP. For an overview of the SDK and instructions on installing it, see [Set up the Azure SDK for PHP](app-service-web/web-sites-php-mysql-deploy-use-git.md)
 
 ## Create a web form for making a call
 The following HTML code shows how to build a web page (**callform.html**) that retrieves user data for making a call:
 
-    <html>
-    <head>
-        <title>Automated call form</title>
-    </head>
-    <body>
-    <h1>Automated Call Form</h1>
-     <p>Fill in all fields and click <b>Make this call</b>.</p>
-      <form action="makecall.php" method="post">
-       <table>
-         <tr>
-               <td>To:</td>
-               <td><input type="text" size=50 name="callTo" value=""></td>
-         </tr>
-         <tr>
-               <td>From:</td>
-               <td><input type="text" size=50 name="callFrom" value=""></td>
-         </tr>
-         <tr>
-               <td>Call message:</td>
-               <td><input type="text" size=100 name="callText" value="Hello. This is the call text. Good bye." /></td>
-         </tr>
-         <tr>
-               <td colspan=2><input type="submit" value="Make this call"></td>
-         </tr>
-       </table>
-     </form>
-     <br/>
-    </body>
-    </html>
+```html
+<html>
+<head>
+    <title>Automated call form</title>
+</head>
+<body>
+<h1>Automated Call Form</h1>
+ <p>Fill in all fields and click <b>Make this call</b>.</p>
+  <form action="makecall.php" method="post">
+   <table>
+     <tr>
+           <td>To:</td>
+           <td><input type="text" size=50 name="callTo" value=""></td>
```

Comment:
@joliveros the original code tried to indent HTML to 2 spaces. ^^ (also don't forget the `<title>`)